### PR TITLE
Content changes  affecting UK and Jordan resident User

### PR DESCRIPTION
**Pull request by ikennaokpala**
[Trello card](https://trello.com/c/B2qDmbTl/145-jordan-marriage-tool-update-content-change-request)

Content changes affecting:

- opposite sex outcomes where user is in UK
- opposite sex outcomes where user is in Jordan

## Factcheck

[Preview Link](https://smart-answers-pr-2560.herokuapp.com/marriage-abroad/y/jordan/ceremony_country/partner_british/opposite_sex)
[GOV.UK](https://gov.uk/marriage-abroad/y/jordan/ceremony_country/partner_british/opposite_sex)

### Expected changes

* Removal of 21 day notice for uk and Jordanian residents
* Removal of birth and naturalisation certificates

### Before

screen shot 2016-06-02 at 12 06 37

### After

screen shot 2016-06-02 at 12 06 50

#### Affected pages

https://smart-answers-pr-2560.herokuapp.com/marriage-abroad/y/jordan/ceremony_country/partner_british/opposite_sex
https://smart-answers-pr-2560.herokuapp.com/marriage-abroad/y/jordan/ceremony_country/partner_local/opposite_sex
https://smart-answers-pr-2560.herokuapp.com/marriage-abroad/y/jordan/ceremony_country/partner_other/opposite_sex
https://smart-answers-pr-2560.herokuapp.com/marriage-abroad/y/jordan/uk/partner_british/opposite_sex
https://smart-answers-pr-2560.herokuapp.com/marriage-abroad/y/jordan/uk/partner_local/opposite_sex
https://smart-answers-pr-2560.herokuapp.com/marriage-abroad/y/jordan/uk/partner_other/opposite_sex

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_uk_or_ceremony_country.govspeak.erb [hunk 1]
```diff
@@ -47,7 +47,9 @@
     You’ll be asked to provide a certificate of no impediment (CNI) to prove you’re allowed to marry. Contact the local notary public to check if you need a CNI.
 
   <% elsif calculator.ceremony_country == 'jordan' %>
-    You need to have been living in <%= calculator.country_name_lowercase_prefix %> for 21 days.
+    <% unless calculator.resident_of_uk? || calculator.resident_of_ceremony_country? %>
+      You need to have been living in <%= calculator.country_name_lowercase_prefix %> for 21 days.
+    <% end %>
```
Comment (chrisroos) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_uk_or_ceremony_country.govspeak.erb]: I believe you can get rid of this section completely. This outcome is only rendered for residents of the UK or the ceremony country, which means that either `calculator.resident_of_uk?` or `calculator.resident_of_ceremony_country?` will be true; which in turn means that the content in the `unless` block will never be shown.

**Comment by chrisroos** [comment]
Aside from my comment about removing the unnecessary section, this looks good to me.

**Comment by sistephens** [comment]
Content LGTM 👍

**Comment by ikennaokpala** [comment]
@sistephens thanks for the feedback

**Pull request merged**